packages/cloud-cognitive/src/components/WebTerminal/WebTerminal.js

```diff
@@ -146,7 +146,7 @@ export let WebTerminal = React.forwardRef(
             hasIconOnly
             renderIcon={Close}
             kind="ghost"
-            iconDescription={closeIconDescription}
+            iconDescription="Close web terminal"
```

Comment:
why did you remove the `closeIconDescription` prop?

Reply:
Thinking about it, what else can this be called? The literal action is closing the web terminal, I don't think designers would really want to change this to something else? Thoughts?